rfcs/convergence/authoring-stories.md

```diff
@@ -373,26 +357,15 @@ export const ButtonWithIcon = () => <Button icon={<CalendarIcon />}>Text</Button
 
 ### 8. location and naming convention
 
-1. Story files should be in the same folder as the component which they are meant to show off. This will usually be the component which is the most unique to the story.
-2. Name of every .stories.tsx file should start with name of a component which it is targeting, to improve colocation in alphabetical file ordering. If appropriate, story name should be adjusted via configuration to a well readable name, suitable for documentation, like this:
-
-```tsx
-export const ButtonPrimary = (props: ButtonProps) => <Button {...props}>Text</Button>;
-ButtonPrimary.args = {
-  primary: true,
-};
-ButtonPrimary.storyName = 'Better story name';
-```
-
-3. If an internal story is broken out into its own file, then the file should mirror the name of the story, which will include the `Internal` keyword and a `.internal.stories` suffix for easier IDE search.
+All `.stories.tsx` which are related to a single docs page files must be in the same folder, which can either be the folder with component code, or a folder called `stories`.
 
 ### 9. UX of stories
 
 #### default story
 
 Every component must have a story called `Default`, which:
 
-- must be defined as first story inline or as first re-expored story when using multi-file pattern - see [chapter 7](#7-dissecting-big-story-files-into-smaller-ones) for examples
+- must be re-expored as the first story - see [chapter 7](#7-dissecting-big-story-files-into-smaller-ones) for examples
```

Comment:
Typo, plus I'm not sure what "as the first story" means in the context of multiple files.
```suggestion
- must be re-exported as the first story - see [chapter 7](#7-dissecting-big-story-files-into-smaller-ones) for examples
```

Reply:
Thanks! "re-exported as the first story" means, that the export of `Default` in `Button.stories.tsx` must come before any other named export.

Good:

 ```tsx
 // @filename Button.stories.tsx
import { Button } from './Button'; // the component
import { Meta } from '@storybook/react';

// 💡 `Default` re-export needs to be always first !
export * from 'ButtonDefault.stories';
export * from 'ButtonWithIcon.stories';

export default {
  title: 'Components/Button',
  component: Button,
} as Meta;
```

Bad:
 ```tsx
 // @filename Button.stories.tsx
import { Button } from './Button'; // the component
import { Meta } from '@storybook/react';

export * from 'ButtonWithIcon.stories';
export * from 'ButtonDefault.stories'; // Default story is exported second 🙅

export default {
  title: 'Components/Button',
  component: Button,
} as Meta;
```

Can you think of a way to make the RFC clearer?